backend/src/main/java/atlantbh/restaurants/services/UserService.java

```diff
@@ -53,11 +51,14 @@ public LoginResponseDTO checkLoginData(LoginRequestDTO request) {
     }
 
     public Boolean isEmailTaken(String email) {
-        Criteria criteria = repository.getBaseCriteria();
-        criteria.add(Restrictions.eq("email", email));
-        if (criteria.list().size() > 0)
-            return true;
-        return false;
+        try {
+            if (repository.isEmailTaken(email)) {
```

Comment:
`return repository.isEmailTaken(email)` 🤐

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb